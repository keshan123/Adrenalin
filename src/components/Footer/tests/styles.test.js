import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import {
  FooterContainer,
  LogoContainer,
  NavigationOptionsContainer,
  NavigationItem,
} from '../styles';

test('FooterContainer renders', () => {
  const tree = renderer.create(
      <FooterContainer/>
  ).toJSON()
  expect(tree).toMatchSnapshot()
})

test('LogoContainer renders', () => {
  const tree = renderer.create(
      <LogoContainer/>
  ).toJSON()
  expect(tree).toMatchSnapshot()
})

test('NavigationOptionsContainer renders', () => {
  const tree = renderer.create(
      <NavigationOptionsContainer/>
  ).toJSON()
  expect(tree).toMatchSnapshot()
})

test('NavigationItem renders', () => {
  const tree = renderer.create(
      <NavigationItem/>
  ).toJSON()
  expect(tree).toMatchSnapshot()
})
