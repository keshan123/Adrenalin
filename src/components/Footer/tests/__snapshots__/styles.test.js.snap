// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterContainer renders 1`] = `
.c0 {
  -webkit-align-items: flex-end;
  -webkit-box-align: flex-end;
  -ms-flex-align: flex-end;
  align-items: flex-end;
  border-top: 1px solid black;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  height: 40px;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  margin: 100px 93px 53px 93px;
}

@media (max-width:768px) {
  .c0 {
    margin: 100px 55px 53px 55px;
  }
}

<div
  className="c0"
/>
`;

exports[`LogoContainer renders 1`] = `
.c0 {
  background-image: url('require(../assets/adrenalin.svg'));
}

<div
  className="c0"
/>
`;

exports[`NavigationItem renders 1`] = `
.c0 {
  color: #000000;
  font-family: Arial,Helvetica Neue,Helvetica,sans-serif;
  font-size: 13px;
  font-weight: regular;
  margin-left: 5px;
}

<div
  className="c0"
/>
`;

exports[`NavigationOptionsContainer renders 1`] = `
.c0 {
  color: #000000;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  font-family: ArialMT;
  font-size: 13px;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  -webkit-letter-spacing: 0;
  -moz-letter-spacing: 0;
  -ms-letter-spacing: 0;
  letter-spacing: 0;
  line-height: 14px;
  width: 436px;
}

<div
  className="c0"
/>
`;
